sponsorship_compassion/wizards/generate_gift_wizard.py

```diff
@@ -85,6 +85,7 @@ def generate_invoice(self):
                 invoice_date = self.invoice_date
             inv_data = self._setup_invoice(contract, invoice_date)
             invoice = self.env["account.invoice"].create(inv_data)
+            invoice.partner_bank_id = contract.partner_id.bank_ids[0].id
```

Comment:
This has the risk of failing if there are no bank accounts on partner (which could be the case for non LSV/DD sponsors). 
```suggestion
            invoice.partner_bank_id = contract.partner_id.bank_ids[:1].id
```

Reply:
Nice catch ! Thank you